external/storm-hdfs/src/test/java/org/apache/storm/hdfs/spout/TestHdfsSemantics.java

```diff
@@ -124,7 +125,7 @@ public void testAppendSemantics() throws Exception {
 
         //2 try to append to a closed file
         try (FSDataOutputStream os2 = fs.append(file1)) {
-            assertThat(os2, notNull());
+            assertThat(os2, notNullValue());
```

Comment:
The tests weren't running, so this has always been wrong. Switching to the right matcher fixes the test.